src/ast.rs

```diff
@@ -7791,7 +7795,7 @@ mod tests {
                                         SourceName(Identifier {
                                             start: 3,
                                             end: 6,
-                                        }))))), None),
+                                        }))))), Vec::new()),
```

Comment:
I think we need `vec![]` rather than `Vec::new()`, to satisfy the `no-std` requirement? That's where the CI is failing.

Reply:
Good point. I was wondering about CI already, it seems that external contributors cannot see CI...

![image](https://user-images.githubusercontent.com/1433023/79392458-b56ce280-7f73-11ea-9cc3-b5350e59d279.png)

Reply:
I can't see it through there either for some reason, but going [here](https://travis-ci.org/github/gimli-rs/cpp_demangle) does the trick.

Reply:
travis notifications have been flaky in gimli as well.

Reply:
I wonder if this started happening after the project was moved to gimli-rs? Could be an old webhook, potentially.